instance/instance.go

```diff
@@ -154,13 +176,20 @@ func (s *Store) Update(w http.ResponseWriter, r *http.Request) {
 		var editionDoc *models.Edition
 
 		datasetID := currentInstance.Links.Dataset.ID
-		instance.Links.Job = currentInstance.Links.Job
+		if instance.Links != nil {
```

Comment:
Should this be `if instance.Links == nil {` ? if there are links provided in the instance, why would they be overwritten by the currentInstance.Links?

Reply:
Actually this can be removed altogether by reworking the logic around spatial links.